package.json

```diff
@@ -2,6 +2,7 @@
   "name": "pc-nrfconnect-app-boilerplate",
```

Comment:
Should this perhaps be pc-nrfconnect-boilerplate, as that is the name of the repo?